website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,56 @@
+``
+
+- The language syntax and semantics for existing language elements will not change within a major version.
+- A spec that was building successfully should build successfully with the new version of the compiler and libraries with the same major version.
+- New types and functionalities can be added in a minor version release.
+  - If a spec updates to use that new functionality/type it is not considered a breaking change if it goes through a library/emitter that will now break.
+- TypeScript types might change in ways that introduce compilation type checking errors. Those are not considered breaking changes.
+
+### Bugs
+
+A bug fix that requires breaking will not be considered a breaking change if the behavior was already broken to start with.
+
+## Categories of Breaking Changes
+
+- <Badge text="Level 0" variant="danger" />: Those Apis shouldn't be broken unless we make a major
+  version and a lot of consideration should given in making any of those. This should be apis that
+  are recommended to be used by library author.
+- <Badge text="Level 1" variant="note" />: Those describe change that are not semantically relevant
+  but still produce noticeable changes in the output. This could be for example a change in the way
+  the output is formatted.
+- <Badge text="Level 2" variant="tip" />: Those are internal apis that could be accessed but
+  touching it would be at the library author risk and breaking change wouldn't even need to be
+  documented
+
+| Example                          | Level                               | Description                                                                                                                                                                                                                                                                                       |
+| -------------------------------- | ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TypeKit                          | <Badge text="0" variant="danger" /> | TypeKits are the recommend way to expose apis for a library. Those should be extremely stable.                                                                                                                                                                                  |
+| Additional helpers for a library | <Badge text="0" variant="danger" /> | Depending on the helper this could be considered critical or more advanced usage                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| Additional library helper functions | <Badge text="0" variant="danger" /> | Helper and metadata accessor functions exported from the public surface of the compiler or a TypeSpec library are generally considered stable unless otherwise specified or only exported through an experimental subpath.                                                                                                                                                                                                               |
```